<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="bt_zoom_in">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>30</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: Outset;
font: 900 20pt &quot;Arial Black&quot;;
border-width: 1px;
border-radius: 90px;</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_zoom_out">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>30</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: Outset;
font: 900 20pt &quot;Arial Black&quot;;
border-width: 1px;
border-radius: 90px;
</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>980</width>
     <height>660</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_imprimir">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>140</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>    Imprimir</string>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_sair">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>310</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>SAIR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>30</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string> ZOOM</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
